comparison/comparison.md

```diff
@@ -0,0 +1,375 @@
+# Comparison with PyTorch's DataLoader
+
+```python
+import lazy_dataset
+from torch.utils import data as torch_data
+from torch.utils.data import DataLoader as TorchDataLoader
+```
+
+| Example | Feature | `lazy_dataset` | `TorchDataLoader` |
+| ------- | ------- | :------------: | :---------------: |
+|    1    | Dataset class | `lazy_datsaset.Dataset`| `torch_data.Dataset` |
+|    2    | Batching | yes | yes |
+|    2    | Collate batch | manual | automatic |
+|    3    | Shuffle | yes | yes |
+|    4    | Multi-process data loading | yes | yes |
+|    see [README](../README.md)    | Filter (builtins.filter) | yes | no |
+|    5    | Sort by key | yes | no |
```

Comment:
Can you add `sort by value:  yes no`?

Reply:
Considered in commit https://github.com/fgnt/lazy_dataset/pull/27/commits/30012f7a71712eba54a06745121c2b3f399189c5